components/navbar/Navbar.js

```diff
@@ -5,8 +5,10 @@ import NavLink from "./NavLink";
 import Link from "../Link";
 import app from "../../config/app.json";
 import Image from "next/legacy/image";
-import { FaGithub } from "react-icons/fa";
-import { IconContext } from "react-icons";
+import getIcon from "../Icon";
+import { IconContext } from "react-icons/lib/cjs/iconContext";
```

Comment:
Changing this import throws an error for using `value` for styles

<img width="552" alt="Screenshot 2023-02-04 at 14 54 57" src="https://user-images.githubusercontent.com/624760/216755998-22e746cb-2711-49e4-b2bf-21087dd5d3a6.png">

Reply:
Ugh, a hydration error. I will have a look at it.

Reply:
thank you 👍

Reply:
I removed IconContext & added a tailwind class to fix the problem.
FYI, vertical-align is irrelevant for a svg element so I didn't add it with tailwind.

Reply:
oh I thought the `IconContext` was needed to add styles to the icon (not sure why I thought that)

Thanks for fixing, I will have a quick play and merge in the morning

Reply:
`IconContext` is only needed if something needs to be applied directly to the svg & can't be applied to a wrapper element.
In this case because the svg sets `stroke` & `fill` to `currentColor` then it will use the inherited color.

Reply:
oh I see, thank you 👍